File: Example_GunaAnimateWindow/Example_GunaAnimateWindow/Form1.vb
Public Class Form1
    Private Sub GunaButton1_Click(sender As Object, e As EventArgs) Handles GunaButton1.Click
        Dim f As Form2 = New Form2
        f.GunaAnimateWindow1.AnimationType = Guna.UI.WinForms.GunaAnimateWindow.AnimateWindowType.AW_CENTER
        f.Show()
    End Sub

    Private Sub GunaButton2_Click(sender As Object, e As EventArgs) Handles GunaButton2.Click
        Dim f As Form2 = New Form2
        f.GunaAnimateWindow1.AnimationType = Guna.UI.WinForms.GunaAnimateWindow.AnimateWindowType.AW_BLEND
        f.Show()
    End Sub

    Private Sub GunaButton3_Click(sender As Object, e As EventArgs) Handles GunaButton3.Click
        Dim f As Form2 = New Form2
        f.GunaAnimateWindow1.AnimationType = Guna.UI.WinForms.GunaAnimateWindow.AnimateWindowType.AW_HOR_NEGATIVE
        f.Show()
    End Sub

    Private Sub GunaButton4_Click(sender As Object, e As EventArgs) Handles GunaButton4.Click
        Dim f As Form2 = New Form2
        f.GunaAnimateWindow1.AnimationType = Guna.UI.WinForms.GunaAnimateWindow.AnimateWindowType.AW_HOR_POSITIVE
        f.Show()
    End Sub

    Private Sub GunaButton5_Click(sender As Object, e As EventArgs) Handles GunaButton5.Click
        Dim f As Form2 = New Form2
        f.GunaAnimateWindow1.AnimationType = Guna.UI.WinForms.GunaAnimateWindow.AnimateWindowType.AW_VER_NEGATIVE
        f.Show()
    End Sub

    Private Sub GunaButton6_Click(sender As Object, e As EventArgs) Handles GunaButton6.Click
        Dim f As Form2 = New Form2
        f.GunaAnimateWindow1.AnimationType = Guna.UI.WinForms.GunaAnimateWindow.AnimateWindowType.AW_VER_POSITIVE
        f.Show()
    End Sub
End Class

File: Example_GunaAnimateWindow/ExampleCsharp_GunaAnimateWindow/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;

namespace ExampleCsharp_GunaAnimateWindow
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void GunaButton1_Click(object sender, EventArgs e)
        {
            Form2 f = new ExampleCsharp_GunaAnimateWindow.Form2();
            f.gunaAnimateWindow1.AnimationType = Guna.UI.WinForms.GunaAnimateWindow.AnimateWindowType.AW_CENTER ;
            f.Show();

        }

        private void GunaButton2_Click(object sender, EventArgs e)
        {
            Form2 f = new ExampleCsharp_GunaAnimateWindow.Form2();
            f.gunaAnimateWindow1.AnimationType = Guna.UI.WinForms.GunaAnimateWindow.AnimateWindowType.AW_BLEND;
            f.Show();
        }

        private void GunaButton3_Click(object sender, EventArgs e)
        {
            Form2 f = new ExampleCsharp_GunaAnimateWindow.Form2();
            f.gunaAnimateWindow1.AnimationType = Guna.UI.WinForms.GunaAnimateWindow.AnimateWindowType.AW_HOR_NEGATIVE ;
            f.Show();
        }

        private void GunaButton4_Click(object sender, EventArgs e)
        {
            Form2 f = new ExampleCsharp_GunaAnimateWindow.Form2();
            f.gunaAnimateWindow1.AnimationType = Guna.UI.WinForms.GunaAnimateWindow.AnimateWindowType.AW_HOR_POSITIVE ;
            f.Show();
        }

        private void GunaButton5_Click(object sender, EventArgs e)
        {
            Form2 f = new ExampleCsharp_GunaAnimateWindow.Form2();
            f.gunaAnimateWindow1.AnimationType = Guna.UI.WinForms.GunaAnimateWindow.AnimateWindowType.AW_VER_NEGATIVE ;
            f.Show();
        }

        private void GunaButton6_Click(object sender, EventArgs e)
        {
            Form2 f = new ExampleCsharp_GunaAnimateWindow.Form2();
            f.gunaAnimateWindow1.AnimationType = Guna.UI.WinForms.GunaAnimateWindow.AnimateWindowType.AW_VER_POSITIVE;
            f.Show();
        }
    }
}
